src/services/resources/controllers.ts

```diff
@@ -43,6 +43,18 @@ export const healthCheck = async (req: Request, res: Response) => {
     return sendResponse(res, response);
 };
 
+export const volunteers = async (req: Request, res: Response) => {
+    const { captain_id } = req.params;
+
+    if(captain_id === undefined){
+        const response = serverError("Captain_ID is required");
```

Comment:
This should be a "badRequestError" because this means the front end implementation is incorrect - server error is like the server crashed unexpectedly

Reply:
Added